[Android] Fix for Bullet Overlay

molecula451 opened the pull request:
So far we're still having fun with our style elements it was the time for our cursor-overlay element <code> .android .bullet-cursor-overlay </code> since it is our state class for our Android elements.

Even though our properties are very well rendering our bullets and positioning. The <code> overlay</code> itself was not measure according to our needs.

### Actual issue: #211 

![70553055-d5694100-1b50-11ea-8a90-95e4b91f18d1](https://user-images.githubusercontent.com/41552663/70766880-9b0cc900-1d35-11ea-9bea-0d49756f20af.png)

### Expected Behaviour:

![overlay](https://user-images.githubusercontent.com/41552663/70770215-e5487700-1d42-11ea-9b09-3db646800339.jpg)


- [x] Bullet diameter should be about equal to half the height of a capital letter, as shown.
- [x] Bullet overlay diameter should be slightly less than the full height of a capital letter, as shown.
- [x] The spacing between the bullet and the text may need to adjusted to compensate for the larger bullet size. The relative spacing can be seen in the reference screenshot.
- [x] These requirements must be made for all six bullet types.

molecula451 commented:
@raineorshine How it's this one looking for you?

raineorshine commented:
Thanks! Can you reference the issue number, please?

> So far we're still having fun with our style elements it was the time for our cursor-overlay element .android .bullet-cursor-overlay since it is our state class for our Android elements.

> Even tough our properties are very well rendering our bullets and positioning the overlay itself was not measure according to our needs.

I'm having trouble understanding your English here. Can you try to clarify?

raineorshine commented:
Please send screenshots when you believe all requirements are complete. Do not send screenshots for incomplete work unless you are blocked and have questions about how to move on. Thanks!

molecula451 commented:
> Thanks! Can you reference the issue number, please?
> 
> > So far we're still having fun with our style elements it was the time for our cursor-overlay element .android .bullet-cursor-overlay since it is our state class for our Android elements.
> 
> > Even tough our properties are very well rendering our bullets and positioning the overlay itself was not measure according to our needs.
> 
> I'm having trouble understanding your English here. Can you try to clarify?

Even though* , tiny mistypo. Issue already referenced

molecula451 commented on src/App.css:
```diff
@@ -753,34 +758,50 @@ h1 .num-contexts {
+  opacity: 1;
+}
+
+/* Our Awesome first child (overlay) independent with its own properties */
```
Some new comments added for code maintenance.

molecula451 commented:
- Added .android classes for <code> .bullet .glyph </code> and <code> .bullet-cursor-overlay   </code > and fixed.

Hi Raine. Hope you are doing great. I took a chance on this one to re-factor and applied some new properties for the enhacement.

The overlay so far it's now rendering as expected in both. Portrait (Small phones) Aswell as on tablet.

We have: 

**Arrow Down context view activated:**

![2019_12_19_11 49 31_edit](https://user-images.githubusercontent.com/41552663/71222396-d0219a00-22a6-11ea-91a9-29a6ae538724.jpg)

**Closed Arrow Down**
![2019_12_19_11 48 37_edit](https://user-images.githubusercontent.com/41552663/71222420-ec253b80-22a6-11ea-961a-33a3b2c08007.jpg)

**Open Circle Context View** activated (I am a tiny doubtful about this one but IMO it is an pretty well looking this way)

![2019_12_19_11 47 58_edit](https://user-images.githubusercontent.com/41552663/71222462-09f2a080-22a7-11ea-93b5-6094c2b6f600.jpg)

**Closed Circle** (This snap it's somehow extra zoomed )
![2019_12_19_11 47 32_edit](https://user-images.githubusercontent.com/41552663/71222518-3c9c9900-22a7-11ea-89a1-9673c7b855fc.jpg)

We can't see (Overlays) when our bullets are not touched 

- Open Arrow Right
- Open Circle (Not touching)
- Closed Circle (Not touching

But the alignment still on a well good fit.

![2019_12_19_11 50 26_edit](https://user-images.githubusercontent.com/41552663/71222869-7d48e200-22a8-11ea-8f99-f5e978532b5b.jpg)

raineorshine commented:
This looks great! Thanks for your persistence and careful attention to detail.

Please resolve the merge conflicts and then I will accept.

molecula451 commented:
> This looks great! Thanks for your persistence and careful attention to detail.
> 
> Please resolve the merge conflicts and then I will accept.

I believe it's pretty much done Raine.
